modules/javafx.base/src/main/java/javafx/collections/ModifiableObservableListBase.java

```diff
@@ -29,6 +29,7 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.ListIterator;
+import java.util.Objects;
```

Comment:
This import seems unused, please remove.

Reply:
Indeed. Removed it.